task/acs-image-check/0.1/acs-image-check.yaml

```diff
@@ -128,3 +135,17 @@ spec:
       script: |
         #!/usr/bin/env bash
         cat /steps-shared-folder/acs-image-check.json
+
+    - name: fail-if-rox-image-check-failed
+      image: registry.access.redhat.com/ubi8-minimal@sha256:d16d4445b1567f29449fba3b6d2bc37db467dc3067d33e940477e55aecdf6e8e
+      script: |
+        #!/usr/bin/env bash
+        ROX_IMAGE_CHECK_EXIT_CODE_PATH=$(steps.step-rox-image-check.exitCode.path)
```

Comment:
Shouldn't there be something like `onError: continue` on the rox-image-check step?